packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -223,8 +231,10 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
             return;
         }
         // Line validation
-        const favoriteDataSetPattern = /^\[.+\]\:\s[a-zA-Z#@\$][a-zA-Z0-9#@\$\-]{0,7}(\.[a-zA-Z#@\$][a-zA-Z0-9#@\$\-]{0,7})*\{p?ds\}$/;
-        const favoriteSearchPattern = /^\[.+\]\:\s.*\{session}$/;
+
+        // eslint-disable-next-line no-useless-escape
```

Comment:
Was the ESLint warning here a false positive?

Reply:
Yep - although the dollar sign wouldn't need to be escaped in a regular string, ESlint interprets the regex as a regular string. `$` would be interpreted as end of string for regex, so this is a false positive 🙂

Reply:
A `$` inside square brackets is evaluated as a literal character rather than end of string, so I think ESLint is valid in saying that we could remove the escape character.

Reply:
Ah - didn't realize that regex has different behavior for when the `$` is inside square brackets. Thanks for the clarification, will update 👍